gaurun/conf.go

```diff
@@ -124,3 +127,46 @@ func ConfigHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Server", serverHeader())
 	fmt.Fprintf(w, b.String())
 }
+
+func ConfigPushersHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
```

Comment:
> should be POST or PUT?

In HTTP specification, PUT is idenpotent but POST is not. So PUT is better.

> GET just returns a current pusher value

The reponse of `GET /config/app` and `GET /stat/app` includes this value. Instead you can use them.